<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>742</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Palatino Linotype</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">alternate-background-color: rgb(167, 167, 167);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="font">
       <font>
        <family>Palatino Linotype</family>
       </font>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="FaceDetection">
       <attribute name="title">
        <string>Face Detection</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_8">
        <item row="1" column="0">
         <layout class="QGridLayout" name="gridLayout_1">
          <item row="0" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_1">
            <item>
             <widget class="QGroupBox" name="imageBox_1">
              <property name="font">
               <font>
                <family>Palatino Linotype</family>
                <pointsize>10</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="title">
               <string>Input</string>
              </property>
              <widget class="QPushButton" name="button_1">
               <property name="geometry">
                <rect>
                 <x>170</x>
                 <y>450</y>
                 <width>171</width>
                 <height>51</height>
                </rect>
               </property>
               <property name="font">
                <font>
                 <family>Palatino Linotype</family>
                 <pointsize>12</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Detect</string>
               </property>
              </widget>
              <widget class="QLabel" name="image_1">
               <property name="geometry">
                <rect>
                 <x>90</x>
                 <y>60</y>
                 <width>370</width>
                 <height>320</height>
                </rect>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="imageBox_2">
              <property name="font">
               <font>
                <family>Palatino Linotype</family>
                <pointsize>10</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="title">
               <string>Output</string>
              </property>
              <property name="flat">
               <bool>false</bool>
              </property>
              <property name="checkable">
               <bool>false</bool>
              </property>
              <widget class="QLabel" name="image_2">
               <property name="geometry">
                <rect>
                 <x>90</x>
                 <y>60</y>
                 <width>370</width>
                 <height>320</height>
                </rect>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
              <widget class="QLabel" name="label_1">
               <property name="geometry">
                <rect>
                 <x>80</x>
                 <y>400</y>
                 <width>281</width>
                 <height>81</height>
                </rect>
               </property>
               <property name="font">
                <font>
                 <family>Palatino Linotype</family>
                 <pointsize>10</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Computation Time:</string>
               </property>
               <property name="scaledContents">
                <bool>false</bool>
               </property>
               <property name="textInteractionFlags">
                <set>Qt::NoTextInteraction</set>
               </property>
              </widget>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="browse_0">
          <property name="font">
           <font>
            <family>Palatino Linotype</family>
            <pointsize>10</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Browse</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="FaceRecognization">
       <attribute name="title">
        <string>Face Recognization</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QGroupBox" name="imageBox_3">
            <property name="font">
             <font>
              <family>Palatino Linotype</family>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="title">
             <string>Input</string>
            </property>
            <widget class="QPushButton" name="button_2">
             <property name="geometry">
              <rect>
               <x>170</x>
               <y>450</y>
               <width>161</width>
               <height>51</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <family>Palatino Linotype</family>
               <pointsize>12</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Recognize</string>
             </property>
            </widget>
            <widget class="QLabel" name="image_3">
             <property name="geometry">
              <rect>
               <x>90</x>
               <y>60</y>
               <width>370</width>
               <height>320</height>
              </rect>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="imageBox_4">
            <property name="font">
             <font>
              <family>Palatino Linotype</family>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="title">
             <string>Output</string>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
            <property name="checkable">
             <bool>false</bool>
            </property>
            <widget class="QLabel" name="label_2">
             <property name="geometry">
              <rect>
               <x>80</x>
               <y>400</y>
               <width>281</width>
               <height>81</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <family>Palatino Linotype</family>
               <pointsize>10</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Computation Time:</string>
             </property>
             <property name="scaledContents">
              <bool>false</bool>
             </property>
             <property name="textInteractionFlags">
              <set>Qt::NoTextInteraction</set>
             </property>
            </widget>
            <widget class="QLabel" name="image_4">
             <property name="geometry">
              <rect>
               <x>90</x>
               <y>60</y>
               <width>370</width>
               <height>320</height>
              </rect>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="browse_1">
          <property name="font">
           <font>
            <family>Palatino Linotype</family>
            <pointsize>10</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Browse</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Evauation">
       <attribute name="title">
        <string>Evaluation</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QGroupBox" name="imageBox_5">
              <property name="font">
               <font>
                <family>Palatino Linotype</family>
                <pointsize>10</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="title">
               <string>ROC</string>
              </property>
              <widget class="QLabel" name="image_5">
               <property name="geometry">
                <rect>
                 <x>280</x>
                 <y>100</y>
                 <width>491</width>
                 <height>291</height>
                </rect>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
              <widget class="QPushButton" name="button_3">
               <property name="geometry">
                <rect>
                 <x>480</x>
                 <y>460</y>
                 <width>111</width>
                 <height>51</height>
                </rect>
               </property>
               <property name="font">
                <font>
                 <family>Palatino Linotype</family>
                 <pointsize>11</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>View</string>
               </property>
              </widget>
              <widget class="QLabel" name="label_3">
               <property name="geometry">
                <rect>
                 <x>370</x>
                 <y>420</y>
                 <width>59</width>
                 <height>18</height>
                </rect>
               </property>
               <property name="font">
                <font>
                 <family>Palatino Linotype</family>
                 <pointsize>10</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Accuracy:</string>
               </property>
               <property name="scaledContents">
                <bool>false</bool>
               </property>
               <property name="textInteractionFlags">
                <set>Qt::NoTextInteraction</set>
               </property>
              </widget>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1080</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="load_image_action">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="load_second_image_action">
   <property name="text">
    <string>Load Second Image</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
